users/tests.py

```diff
@@ -1,3 +1,65 @@
+from rest_framework.test import APIClient
 from django.test import TestCase
 
-# Create your tests here.
+
+class UserRegistrationTestCase(TestCase):
+    def test_user_can_register(self):
+        client = APIClient()
+        response = client.post(
+            '/api/profiles/create/',
+            {'username': 'testuser', 'email': 'testuser@mail.com', 'password': 'testuser_1'},
+            format='json'
+        )
+        self.assertEqual(response.status_code, 201)
```

Comment:
Can you please check the `request.data` ?
Can you please check the user is really saved to database ?